client-participation/js/strings.js

```diff
@@ -110,11 +113,14 @@ preloadHelper.acceptLanguagePromise.then(function() {
       languageCode.match(/^pt-BR/)) {
       _.extend(strings, translations.pt_br);
     }
-     else if (languageCode.match(/^he/)) {
+    else if (languageCode.match(/^he/)) {
       _.extend(strings, translations.he);
     }
-     else if (languageCode.match(/^cy/)) {
+    else if (languageCode.match(/^cy/)) {
       _.extend(strings, translations.cy);
+    }
+	else if (languageCode.match(/^gr/)) {
```

Comment:
heh good call on these little catches :) missed one

```suggestion
    else if (languageCode.match(/^gr/)) {
```